### PR TITLE
Py23 travis [WIP]

### PR DESCRIPTION
**Pull request by ChristopherChudzicki**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,6 +12,18 @@ matrix:
     after_success:
       - coveralls
 
+  - language: python
```
Comment (jolyonb) [inline .travis.yml]: This looks like the correct block to me. Can you go and fix the other PR?

Comment (ChristopherChudzicki) [reply .travis.yml]: Sure, or do you want to review this one and close the other? History is slightly cleaner. 

But if you have a bunch of unpublished comments I can go fix the other. 

(If you want to review this branch, I won't mess with travis anymore till after lunch)

Comment (jolyonb) [reply .travis.yml]: I just finished reviewing the other branch...

Comment (ChristopherChudzicki) [reply .travis.yml]: No problem :) I'm going to wait for this to finish before changing travis on the other branch.

**Comment by jolyonb** [comment]
Well, it failed. Looks like you want requirements3.txt, no? Also, you may need to use pip3 instead of pip. But try these things in the other branch ;-)

**Comment by jolyonb** [comment]
Actually, pip seems fine - it just looks like the version of numpy we use in edX isn't compatible with python3 ;-)

**Comment by ChristopherChudzicki** [comment]
> Well, it failed. Looks like you want requirements3.txt, no? 

Ugh, you're right. Well, at least it ran Python 3.7. Fixing requirements3 on the other branch.

**Pull request closed without merging**